.github/workflows/support-workers-build.yml

```diff
@@ -97,6 +102,13 @@ jobs:
                     - "-PreparePaymentMethodForReuseLambda-"
                   fileName: support-workers.jar
                 dependencies: [cfn]
+              support-workers-typescript:
```

Comment:
I'm glad you've distinguished them by their language as it's clearer 👍  (vs the change from product-move-api -> product-switch-api)